### ISSUE TITLE
Showing more possible suggestions

### ISSUE THREAD

**Issue by aarppe** [giellalt/keyboard-crk#15]
While many times the best possible suggestion based on the morpheme completion approach ends up being the first or second ranked one, just as many times there are more plausible suggestions, e.g.

```
hfst-ospell -S -X -n 12 crk-x-predict_morphs-mobile.zhfst
kanstohtat
"kanstohtat" is NOT in the lexicon:
Corrections for "kanstohtat":
ka-nistohtât    27.249649
kâ-nistohtât    27.499458
kanisitohtâto|    28.749804
kânisitohtâto|    29.000414
ka-nisitohtâht    29.749435
ka-nisitohtâto|    29.749443
kâ-nisitohtâht    29.999447
```

In the above, _ka-nistohtâht_, _kâ-nisitohtâht_, _ka-nisitohtâht_, and _kâ-nisitohtâht_ would all be decent suggestions for the input: _kanstohtat_.

It could be a useful feature, if the user could e.g. swipe the suggestion band to the left, in order to see more possible suggestions, such as 4-5 instead of the current 2-3. Perhaps alternatively, the suggestions could be truncated in their initial or early initial part, to make for more room on the suggestion band.

Of course, this would require serious thinking about the best way to implement such a functionality, if at all, before any of the coding work that would be needed.

**Comment by snomos** [comment]
To ensure identical behavior between command line and phone, please install and test using [divvunspell](/divvun/divvunspell), instead of `hfst-ospell`.

**Comment by aarppe** [comment]
Yes, I can use `divvunspell` for testing. though the issue and the example given above does not concern the order of the suggestions, but their number, i.e. how many suggestions can be made available to the user, as it could be desirable to present more than 2-3 suggestions, e.g. 4-5 or perhaps even more.

**Comment by snomos** [comment]
This is not easy. The problem is not a technical one, but one of user interface design. It should be both obvious and intuitive, as well as as simple as possible, preferably requiring only one tap.

For now I would focus on improving the suggestions so that the most likely one is the first or second suggestion in most cases.

**Comment by aarppe** [comment]
The improvements in the above case require contextual information, distinguishing between TA vs. TI verbs. That is neither trivial.

**Comment by aarppe** [comment]
I spent a lot of time trying to figure out, unsuccessfully, how to delete entries to the personal dictionary, until I was told it was a matter of a simple swipe. There may be some functionalities that require knowing something special to access them, and I could imagine that a swipe could be a way to access more suggestions than the screen allows - and that could be a similar feature that one just has to be told to know.
